radicle-node/src/test/environment.rs

```diff
@@ -133,27 +134,28 @@ impl<G: Signer + cyphernet::Ecdh + 'static> Drop for NodeHandle<G> {
 impl<G: Signer + cyphernet::Ecdh> NodeHandle<G> {
     /// Connect this node to another node, and wait for the connection to be established both ways.
     pub fn connect(&mut self, remote: &NodeHandle<G>) -> &mut Self {
+        let local_events = self.handle.events();
+        let remotel_events = remote.handle.events();
```

Comment:
```suggestion
        let remote_events = remote.handle.events();
```
although I like `local`/`remotel` :joy: